docs/content/sips/017-service-chaining.md

```diff
@@ -14,30 +14,26 @@ Created: Feb 21, 2024
 
 In Spin 2.2, a component can make a HTTP request with only a path (e.g. `/api/accounts`).  Such a request will be routed to the same application, but this is done by Spin prepending the host where the application is running (e.g. `https://mysrsbizness.fermyon.app/api/accounts`).  The consequence of this is that requests go out to the network and come back in, including through load balancers etc.  This adds overhead, and in some environnments could count against quota or incur bandwidth charges.
 
-We have [prototyped](https://github.com/fermyon/spin/pull/2229) dispatching these requests directly to the component handling the route, but this was in the context of an implementation detail of Spin.  The requirement for this project is for an application to declare that it _requires_ dispatch to run locally on the node, and for hosts that cannot meet that prerequisite to reject the application.  This allows developers to write applications as a network of chained microservices, without having to be concerned about what happens if the microservices are not co-located.
+_Service chaining_ refers to carrying out HTTP requests in memory, without going off the node, by invoking the receiving Spin component directly instead of via the HTTP server.
+
+We have [prototyped](https://github.com/fermyon/spin/pull/2229) dispatching HTTP requests directly to the component handling the route, but this was in the context of an implementation detail of Spin.  The requirement for this project is for an application to declare that it _requires_ dispatch to run locally on the node, and for hosts that cannot meet that prerequisite to reject the application.  This allows developers to write applications as a network of chained microservices, without having to be concerned about what happens if the microservices are not co-located.
 
 # User experience
 
 ## Expressing a requirement for chaining
 
-To express that the application requires chaining for self-requests, add the following to `spin.toml`:
+To express that the application requires service chaining, add the following to `spin.toml`:
 
 ```toml
-[application.prerequisites]
-service_chaining = "required"
+[application.host_requirements]
```

Comment:
Continuing the great bike shed tradition:

The term "requirement" doesn't necessarily evoke _enabling_ something for me. More importantly, the _absence_ of a particular "requirement" doesn't necessarily imply that a feature is _disabled_.

Reply:
I guess while I'm here arguing about the shed color I should probably suggest that we build a garage instead:

```toml
[application.host_features]
local_service_chaining = "enabled"
```

Reply:
It occurs to me that if we required allowed hosts to explicitly list `spin.internal` (not wildcards) we could get rid of this section entirely.